### PR TITLE
Capture max of sequences in order to update after migration

### PR DESCRIPTION
**Pull request by makalaaneesh**
https://yugabyte.atlassian.net/browse/DB-5927

Implementation:
- The config value debezium.sink.ybexporter.column_sequence.map needs to be provided which will have a value of the format  `"<column1>:<sequence1>,<column2>:<sequence2>"`
- A new class SequenceObjectUpdater that keeps track of the max value for each sequence. Each record is processed by this class.
- The ExportStatus has a reference to the map object and keeps updating the value in the json file.

### CODE REVIEW

File: debezium-server/debezium-server-ybexporter/src/main/java/io/debezium/server/ybexporter/ExportStatus.java [hunk 1]
```diff
@@ -90,6 +91,14 @@ public void updateMode(ExportMode modeEnum) {
         mode = modeEnum;
     }
 
+    public void setSequenceMaxMap(Map<String, Long> sequenceMax){
+        this.sequenceMax = sequenceMax;
+    }
+
+    public Map<String, Long> getSequenceMap(){
```
Comment (amit-yb) [inline debezium-server/debezium-server-ybexporter/src/main/java/io/debezium/server/ybexporter/ExportStatus.java:98]: getSequenceMaxMap?

File: debezium-server/debezium-server-ybexporter/src/main/java/io/debezium/server/ybexporter/SequenceObjectUpdater.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    public void processRecord(Record r){
+        if (r.op.equals("d")){
+            // not processing delete events because the max would have already been updated in the
+            // prior create/update events.
+            // TODO: question: should we also update the max to the second-max value if the row with max is deleted??
+            return;
+        }
+        for(int i = 0; i < r.valueColumns.size(); i++){
+            String fullyQualifiedColumnName = r.t.schemaName+"."+r.t.tableName+"."+r.valueColumns.get(i);
+            if (columnSequenceMap.get(fullyQualifiedColumnName) != null){
```
Comment (amit-yb) [inline debezium-server/debezium-server-ybexporter/src/main/java/io/debezium/server/ybexporter/SequenceObjectUpdater.java:46]: We should avoid building such strings in critical path.
See if we can have a three level map m[schemaName][tableName][columnName] --> max value

File: debezium-server/debezium-server-ybexporter/src/main/java/io/debezium/server/ybexporter/YbExporterConsumer.java [hunk 3]
```diff
@@ -79,6 +82,26 @@ void retrieveSourceType(Config config){
         }
     }
 
+    Map<String, String> retrieveColumnSequenceMap(Config config){
+        String propertyVal = PROP_PREFIX + "column_sequence.map";
```
Comment (amit-yb) [inline debezium-server/debezium-server-ybexporter/src/main/java/io/debezium/server/ybexporter/YbExporterConsumer.java:86]: sequence_columns.map

**Review by amit-yb — APPROVED** [review]
LGTM. Just a few minor comments.

File: debezium-server/debezium-server-ybexporter/src/main/java/io/debezium/server/ybexporter/SequenceObjectUpdater.java (hunk 2 above)
Comment (makalaaneesh) [reply debezium-server/debezium-server-ybexporter/src/main/java/io/debezium/server/ybexporter/SequenceObjectUpdater.java]: Done. Changed it to a map instead.

File: debezium-server/debezium-server-ybexporter/src/main/java/io/debezium/server/ybexporter/YbExporterConsumer.java (hunk 3 above)
Comment (makalaaneesh) [reply debezium-server/debezium-server-ybexporter/src/main/java/io/debezium/server/ybexporter/YbExporterConsumer.java]: the map is of <column>:<sequence>, not <sequence>:<column>, therefore I think column_sequence makes more sense here.

**Pull request merged**